MVPproject/MVPproject/RecipesModule/Presenter/CategoryPresenter.swift

```diff
@@ -7,6 +7,8 @@ protocol CategoryPresenterProtocol: AnyObject {
     var recipes: [Recipe] { get }
     /// Запрос на закрытие категории
     func closeCategory()
+    /// Переход на экран с детальным описанием рецепта
+    func showDetailView(recipe: Recipe)
```

Comment:
```suggestion
    func showRecipeDetails(recipe: Recipe)
```
лучше не привязываться к слову View, название выбирать не из кода, который будем вызывать, а дизайна, что будет показывать